docs/decisions/0004-which-cookiecutters-included.rst

```diff
@@ -0,0 +1,43 @@
+4. Which Cookiecutters in edx-cookiecutters?
+============================================
+
+Status
+------
+
+Draft
+
+
+Context
+-------
+
+* Currently, the only cookiecutters in this repository are the Open edx public python cookiecutters.
+
+* Since the creation of this repo, others have tried adding public non-python cookiecutters to this repository.
+
+* The name of this repository suggests all public edx cookiecutters will be in this repository.
+
+
+Possible Decision 1
+-------------------
+
+edx-cookiecutter should be the location for all public edx cookiecutters.
```

Comment:
I favor this option. It would allow someone a quick view of everything we have and get uptodate quickly

Reply:
I favor a single repo for all of our cookiecutters as well.

Reply:
+1

This is especially important for community members.
If I want to create "a thing", I don't want to search around just to find out which repo I need and then follow per-repo instructions on how to create the thing. 

We encourage community contribution when we make it easier to find and do what you want (here, templates centrally located).

Also, most Open edX contributing teams are considerably smaller than us; they can't all afford to overly specialize re: frontend-vs-backend development, so for them, excluding non-Python templates here is more of an artificial distinction than it is for us.

Reply:
+1 for having this repo be a central point for public edx cookiecutters.

A question to test the boundaries of this decision:
In the future, if e-commerce or registrar has a cookie-cutter for plugins in their own respective services, does their cookie-cutter go in this repo?

Reply:
That's an interesting use case. I would recommend putting the plugin cookiecutters here, though we might have to change the structure of this repo to make it easier to navigate(especially if we have ton of cookiecuttters in this repo). If they were to make a compelling case of why the plugin cookiecutter doesn't go in this repo, I would ask them to modify the readme to include a link to the cookiecutter. This would allow us to make this repo as the starting point in search of a template for projects.

Reply:
Sounds good.